data/hai/unfettered 0 prologue.txt

```diff
@@ -516,57 +521,992 @@ mission "Unfettered Jump Drive Trading"
+				goto leave
+			label raiders
+			`	"A few eclipses ago they came, in great ships and with superior technology. We, the Hai, had to defend ourselves, so some of us became more military-focused. A lot of those you see here are among those who wanted to repay the raiders in kind once they stopped. Even if our brothers do, we cannot lay down our arms. If these raiders returned, someone has to protect all of us."`
+			label leave
+			`	Someone then calls upon Sookootow in hai and he is forced to promptly leave.`
+
+event "unfettered: access to Mera's lab"
+	planet "Firelode"
+		add outfitter "Mera's Lab"
+
+
+
+outfit "Heshi"
+	category "Unique"
+	cost 10000
+	thumbnail "outfit/unknown"
+	"mass" 1
+	"capture attack" 4
+	"capture defense" 8
+	"maintenance costs" 100
+	description "This is an aquarium containing what the Hai call a 'Heshi' - a huge scorpion looking creature. It must be fed and cared for but can be very useful in damaging the moral of enemies attacking the ship."
+	description "	On the offensive it can be used to intimidate enemies into surrendering, when they are made aware they could become its next meal if they do not cooperate... Furthermore, it is very useful in the event of an interrogation."
+
+# SPOILER: he tests your military aptitude before asking you to take him to pirate space.
+# gameplay wise if you win this mission you should be able to do the following ones, it tests if your strength is high enough.
+# Since it is the first one you can do it many times until you succeed.
+mission "<uhai> 2: Prove Your Strength"
+	autosave
+	repeat
+	deadline 2
+	name `Defeat the captain`
+	description `A young Unfettered captain has provoked you into a duel. Disable his ship and then land back on <destination>. You only have one day, but this duel can happen multiple times.`
+	source "Darkcloak"
+	to offer
+		or
+			has "<uhai> 2: Prove Your Strength: deferred"
+			has "Wanderers: Defend Vara Ke'sok: aborted"
+			random < "Unfettered Jump Drive 1: offered" * 20 + "Unfettered Jump Drive 2: offered" * 25 + "Unfettered Jump Drive 3: offered" * 30 + "Unfettered Jump Drive 4: failed" * 25
+		# 3 is what a default built Sea Scorpion has
+		"armament deterrence" >= 3
+		not "<uhai> 2: Prove Your Strength: done"
+	on offer
+		conversation
+			branch waiting
+				or
+					has "<uhai> 2: Prove Your Strength: deferred"
+					has "<uhai> 2: Prove Your Strength: offered"
+			`You start wandering in the spaceport, and on the way you notice just how much everything around you here is simple, efficient, and ordered, unlike the world of Firelode. The Unfettered present are proudly showing their weapons, more so than on other planets, and some of them are even armored. They seem ready for action, at any time. Thankfully they no longer consider you an enemy, at least for the moment.`
```

Comment:
This line mentions Firelode, but I encountered this mission on my playthrough without ever having visited Firelode. This was pretty confusing.
<img width="192" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/653f26c4-b928-4d81-958c-3e43ecf27419">